infra/cifuzz/run_fuzzers_entrypoint.py

```diff
@@ -26,6 +27,20 @@
     level=logging.DEBUG)
 
 
+def delete_unneeded_docker_images(config):
+  """Deletes unneeded docker images if running in an environment with low
+  disk space."""
+  if not config.low_disk_space:
+    return
+  logging.info('Deleting builder docker images to save disk space.')
+  project_image = docker.get_project_image_name(config.project_name)
+  images = [
+      project_image, 'gcr.io/oss-fuzz-base/base-builder',
+      'gcr.io/oss-fuzz-base/msan-libs-builder'
```

Comment:
should these two constants of gcr.io be in docker.py as well ?

Reply:
Done.